packages/kolibri-zip/generateMimeTypeDB.js

```diff
@@ -45,6 +45,10 @@ const allowedFileExtensions = new Set([
   'md',
   'textile',
   'vtt',
+  // Additional file types for Kolibri
+  'html',
+  'htm',
+  'xhtml',
```

Comment:
I am not sure why these are added. Not that they shouldn't be, but more in the benefit/purpose way

Reply:
As I note in the PR description:
> The HTML and XML utilities were also created in conjunction with work on the draft PR https://github.com/learningequality/kolibri/pull/8070 (making the third use case for the abstraction)

I preferred to include these edits here, as it helps to triangulate the use case for the abstraction.